app/src/main/java/mozilla/org/webmaker/activity/Project.java

```diff
@@ -7,4 +9,16 @@
     public Project() {
         super("project", R.id.project_layout, R.layout.project_layout, R.menu.menu_project);
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
I think @thisandagain's patch will affect this, since it moves some onCreate things around. Worth looking at before we merge to see if there's conflicts or not there